x-pack/plugins/enterprise_search/public/applications/app_search/components/setup_guide/setup_guide.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
```

Comment:
We have a Setup Guide in both shared and in app search, is one of these outdated and unused?

Reply:
No, one's a shared layout that Workplace Search will also be using (https://github.com/elastic/kibana/pull/70979/commits/c6abd302b936a3bd34418a4f4941a58fce47bfbd). The shared component is essentially a layout with the same installation copy, the left pane is per-product marketing copy.

Reply:
Got it, thanks.